src/domain/session/settings/SettingsViewModel.js

```diff
@@ -158,5 +160,28 @@ export class SettingsViewModel extends ViewModel {
             this.emitChange("pushNotifications.serverError");
         }
     }
+
+    get themes() {
+        return Object.keys(this._styleSheetElements);
+    }
+
+    get currentTheme() {
+        for (const name in this._styleSheetElements) {
```

Comment:
The view model should never know about DOM elements, can we have an abstraction over the available and current themes injected through `Platform` here (e.g. `ThemeManager`)?

Reply:
Good call. I had a feeling that this wasn't the best place for this logic. Extracted things into a dedicated `ThemeManager` class stored on the `Platform` instance.